content/tutorials/kubernetes-helm-part-one/index.md

```diff
@@ -0,0 +1,450 @@
+
+  Most of the third-party applications that you want to install on your Kubernetes cluster, like whole monitoring stacks, databases, and other applications, are most likely available as Helm charts.
+
+  On services like [Artifact Hub](https://artifacthub.io/), you can find a lot of Helm charts that you can use to install applications on your Kubernetes cluster.
+
+  The Pulumi Kubernetes provider offers two different ways to install Helm on Kubernetes:
+
+  - Using the `Release` resource
+  - Using the `Chart` resource
+
+  The Helm [Chart](/registry/packages/kubernetes/api-docs/helm/v4/chart) resource renders the templates from your chart and then manages the objects directly with the Pulumi Kubernetes provider.
+
+  The Helm [Release](/registry/packages/kubernetes/api-docs/helm/v4/release) resource uses the Helm SDK to install the Helm chart on your Kubernetes cluster.
+youll_learn:
+- How to install Helm on Kubernetes using Pulumi
+
+estimated_time: 10
+collections_weight: 2
+prereqs:
+- "A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)"
+- "The [Pulumi CLI](/docs/install/)"
+- "A Kubernetes cluster (for example, [kind](https://kind.sigs.k8s.io/))"
+- "[kubectl](https://kubernetes.io/releases/download/#kubectl)"
+- "[helm](https://helm.sh/docs/intro/install/)"
+---
+
+## Never heard of Helm?
+
+What is Helm? Helm is a package manager for Kubernetes and a Helm chart is a collection of different Kubernetes (like Deployment, Service, Ingress, etc.) resources that are bundled together and can be deployed as a unit.
+
+There are many ways to share Helm charts, but the most popular way is by using a Helm repository. Recently Helm supports also OCI registries as a way to share the Helm chart as an OCI artifact.
+
+Additionally, Helm has a templating engine that allows you to customize the Helm chart before you deploy it. It is based on the go templating engine (plus Sprig functions) and allows you to customize the Helm chart based on your needs.
+
+> This templating engine is also the reason why some folks are avoiding Helm. Things can get pretty complex if not taken care.
+
+## Deploying a Helm chart with Pulumi
+
+To start, [login to the Pulumi CLI](/tutorials/cli-authentication/) and create a new Pulumi project. You can use the following command to create a new Pulumi project and select from the list of templates the `helm-kubernetes-<your-programming-language>` template.
+
+```bash
+# Choose your favorite Pulumi supported language
+pulumi new helm-kubernetes-<your-programming-language>
+```
+
+This will create a new Pulumi project with the necessary files to deploy Kubernetes resources and some example resources.
+
+{{< chooser language "typescript,python,go,csharp,yaml" />}}
```

Comment:
Please make sure to incorporate code using the current `example-program` process rather than inline.